index.html

```diff
@@ -233,5 +233,41 @@ <h2 class="aboutMeTitle">
             </div>
         </div>
     </section>
+    <section>
+        <div class="contact">
+            <p class="newProjects">
+                I'm always interested in hearing about 
+                new projects, so if you'd like to chat 
+                please get in touch.
+            </p>
+            <form action="https://formspree.io/f/xoqzkakk" class="contactForm" method="post">
+                <input type="text" id="fname" name="fname" placeholder="Full Name" maxlength="30" required><br>
+                <input type="email" id="email" name="_replyto" placeholder="Enter email" required><br>
+                <textarea name="Opinion" id="Opinion" cols="50" rows="10" maxlength="500" placeholder="Write me something..." required></textarea>
+                <div class="btn-submitContainer">
+                    <button type="submit" id="submit" name="submit" value="submit">Get In Touch</button>
+                </div>
+            </form>       
+        </div>
+    </section>
+    <footer>
+        <img src="images/line3.png" alt="line for footer start">
+        <div>
```

Comment:
- [ ] Kindly make this line occupy the full width of the page. In the design template, the line is from end to end. 

### Your Design

![image](https://user-images.githubusercontent.com/77106962/228265415-5298825b-65ef-4426-847d-9f83a14d1a40.png)

### Figma Template

![image](https://user-images.githubusercontent.com/77106962/228265554-2ecb14b8-ce11-472a-85e4-aafb5c63f967.png)

#### Optional Advice: It would be more semantic to use the `hr` tag since it is a horizontal line